local-setup/jupyter/dataloader/crs_loader.py

```
@@ -0,0 +1,1363 @@
+            import hashlib
+            config_str = json_lib.dumps(new_config, sort_keys=True, separators=(',', ':'))
+
+            # Construct SQL update
+            sql = f"""
+            UPDATE eg_mdms_data
+            SET data = '{json_lib.dumps(new_config)}'::jsonb,
+                lastmodifiedtime = EXTRACT(EPOCH FROM NOW())::bigint * 1000
+            WHERE schemacode = 'tenant.citymodule'
+              AND tenantid = 'pg'
+              AND data->>'code' = '{module_code}';
+            """
```

Ruff (0.15.0)

[error] Possible SQL injection vector through string-based query construction

(S608)